pyproject.toml

```diff
@@ -30,14 +30,15 @@ content-type = "text/markdown"
 
 [project.optional-dependencies]
 all = [
+    "psycopg[binary]",
```

Comment:
Should we retain support for psycopg <3?   I think you can install 2.9 and 3+ at the same time.  Not sure if we need to, but I want to make sure we don't break any existing usage of `sdssdb`.

Reply:
Good point, I had not thought of that. According to [this](https://docs.sqlalchemy.org/en/20/dialects/postgresql.html#module-sqlalchemy.dialects.postgresql.psycopg) SQLA 2.x has native support so maybe that's something we can ensure we test when we update sdssdb to support it.